src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -135,11 +133,18 @@
 
 (def default-graceful-stop-timeout 60000)
 
+(def default-request-idle-timeout
+  1000)
+
 (def default-proxy-idle-timeout
   "The default number of milliseconds to wait before the proxy gives up on the
   upstream server if the :idle-timeout value isn't set in the proxy config."
   60000)
 
+(def default-queue-idle-timeout 60000)
```

Comment:
If the timeout above didn't have a docstring I wouldn't miss having one on this as much, but it does :smile: I think it would be worthwhile to add a docstring here mentioning what it does and that it's in milliseconds.